test/commands/update.nut.ts

```diff
@@ -4,7 +4,7 @@ import * as qq from 'qqjs'
 
 const skipIfWindows = process.platform === 'win32' ? it.skip : it
 
-describe('update', () => {
+describe.skip('update', () => {
```

Comment:
why was this file renamed to `update.nut.ts`?

Reply:
Because it really is a nut test

Reply:
But oclif doesn't have any concept of NUTs - and looking at the package.json it won't ever be run since it doesn't match the regex

Reply:
I left in place just in case we add nuts to oclif. I you prefer I can delete this file.